common/custom-sidebar.js

```diff
@@ -24,6 +24,33 @@ export function updateSidebar(vizConfiguration) {
     return nodeID;
   }
 
+  const getSortedIndex = (nodes, text) => {
+    for (let i = 0; i < nodes.length; i++) {
+      if (nodes[i].textContent.localeCompare(text) > 0) {
+        return i;
+      }
```

Comment:
I would use forEach, to simplify the `let i = 0; i < nodes.length; i++` junk. But whichever you prefer....
```
nodes.forEach((node, i) => {
    if (node.textContent.localeCompare(text) > 0) {
        return i;
    }
};
```